src/modules/feedback/components/FeedbackBoxContainer.js

```diff
@@ -72,7 +73,8 @@ export class FeedbackBoxContainer extends React.Component<PropsType, StateType>
       if (postFeedbackDataOverride) {
         postFeedbackDataOverride(feedbackData)
       } else {
-        await createFeedbackEndpoint(cmsApiBaseUrl).request(feedbackData)
+        const res = await createFeedbackEndpoint(cmsApiBaseUrl).request(feedbackData)
+        if (res) { return res }
```

Comment:
No need to check for the value of the result of the request, it always returns `{}` for the feedback endpoint if it does not fail. Just return `true` if you don't have to catch an error and `false` if you have to. You should also handle `postFeedbackDataOverride` above.

Reply:
changed